src/main/java/com/google/maps/model/OpeningHours.java

```diff
@@ -0,0 +1,58 @@
+package com.google.maps.model;
+
+public class OpeningHours {
+  /**
+   * openNow is a boolean value indicating if the place is open at the current time.
+   */
+  public Boolean openNow;
+
+  static public class Period {
```

Comment:
That sounds like 7 lists of Joda-Periods, each at a Joda-Instant? I was just thinking it'd be great if we could prevent reinventing the wheel (and prevent forcing developers who use this data from writing the same code over & over), but if doing so will be more complex then leave it as it is.